blablas/ep35/index.md

```diff
@@ -0,0 +1,79 @@
+---
+date: 2020-04-12
+time: 20h:30min
+duration: "01:57:56"
+title: "GeeksBlabla : DevC Community"
+tags: ["GeeksBlabla", "DevC", "Community", "Facebook"]
+isNext: false
+video: "2982251615130868"
+published: true
+featured: false
+url: ""
+podcastUrl: https://web.facebook.com/youssouf.elazizi/videos/2982251615130868
+---
+
+In this special episode of GeeksBlabla, some of its core team members discussing what is DevC community, how it starts and what is the relationship between it and Facebook. In the end, the members start revealing some of their secrets and memories they had in the community.	
```

Comment:
Can you please change:
+ Discussing --> discussed
+ How it starts --> how is started
+ What is the relationship... -> how facebook supports devc